### PR TITLE
use component folder name for index.js

### PR DESCRIPTION
**Pull request by erictheswift**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const makeClassNameFromPath = (path) => path.split(process.platform === 'win32' ? "\\" : "/")
```
Comment (chandlerprall) [inline src/utils.js]: using node's `path.sep` instead of detecting the separator slash by platform seems more stable

**Review by chandlerprall — CHANGES_REQUESTED** [review]

File: src/utils.js (hunk 1 above)
Comment (erictheswift) [reply src/utils.js]: wow
missed that path module member (it has been here for ages)

Comment (erictheswift) [reply src/utils.js]: fixed that

**Review by chandlerprall — APPROVED** [review]

**Comment by chandlerprall** [comment]
Thanks! babel-plugin-encapsulate-jsx@3.1.0 has been published to npm

**Pull request merged**